Userland/Applications/PixelPaint/ImageEditor.h

```diff
@@ -173,7 +173,7 @@ class ImageEditor final
 
     Selection m_selection;
 
-    bool m_loaded_from_image {true};
+    bool m_loaded_from_image { true };
```

Comment:
Can you merge this commit with the previous one please? If you don't know how to do that, this video is helpful: https://www.youtube.com/watch?v=ElRzTuYln0M

Reply:
I indeed did not know how to do this. Thanks for the video link.